### PR TITLE
fix link to readme

### PR DESCRIPTION
**Pull request by teallarson**
## What GitHub issue does this PR apply to?

N/A

## What changed and why?

There was a broken link in the docs to the project README.md

## Checklist

- [x] I claimed any associated issue(s) and they are not someone else's
- [x] I have looked at documentation to ensure I made any revisions correctly
- [x] I tested my changes locally to ensure they work
- [x] (If editing Django) I have added or edited any appropriate unit tests for my changes

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -56,24 +57,24 @@ Please make your issue as detailed as possible! Add in the page or URL it happen
...
 ## From Issue to PR
 
-Think you can tackle an issue? Great! 
+Think you can tackle an issue? Great!
 
 1. First, fork the repo so you have your own copy of it
 1. Create a new branch. You can name it whatever you want, but I'd recommend something descriptive, like `issue-123-fix-string-exception`
 1. Make your changes! I suggest committing early and often.
-1. Make sure you test the changes. Check out how to run the project on the [`README.md`](README.md) page.
+1. Make sure you test the changes. Check out how to run the project on the [`README.md`](https://github.com/codethesaurus/codethesaur.us/README.md) page.
```
Comment (geekygirlsarah) [inline CONTRIBUTING.md]: This link gave me a "Not found" error.  I think you want https://github.com/codethesaurus/codethesaur.us/blob/main/README.md .

**Review by geekygirlsarah — CHANGES_REQUESTED** [review]
I think if you fix the README link, this will be good to merge!

**Comment by geekygirlsarah** [comment]
Hey @tealjulia, I figured I'd check in on this since it's been over a week since the last comment. Have any questions or need anything?

**Comment by teallarson** [comment]
Ah, no, just got distracted :)

**Comment by geekygirlsarah** [comment]
That works. Thank you! I'll merge it!

**Pull request merged**